tensorflow/inference/docker/2.0.2/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,84 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    git \
+    wget \
+    vim \
+ && rm -rf /var/lib/apt/lists/*
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+# Add arguments to achieve the version, python and url
+#  PYTHON=python for 2.7
+#  PYTHON=python3 for 3.5, 3.6 is not available directly on 16.04
```

Comment:
Could you clarify this comment? Is 3.6 directly available on 18.04?

Reply:
These aren't my comments. I just copied the old dockerfiles (from 2.0.1) and changed the binary path.